fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java

```diff
@@ -315,19 +313,6 @@ private static CommandLineOptions parseArgs(String[] args) {
         return new CommandLineOptions(false, null);
     }
 
-    // To resolve: "Multiple HTTP implementations were found on the classpath. To avoid non-deterministic
-    // loading implementations, please explicitly provide an HTTP client via the client builders, set
-    // the software.amazon.awssdk.http.service.impl system property with the FQCN of the HTTP service to
-    // use as the default, or remove all but one HTTP implementation from the classpath"
-    // Currently, there are 2 implements of HTTP client: ApacheHttpClient and UrlConnectionHttpClient
-    // The UrlConnectionHttpClient is introduced by #16602, and it causes the exception.
-    // So we set the default HTTP client to UrlConnectionHttpClient.
-    // TODO: remove this after we remove ApacheHttpClient
-    private static void setAWSHttpClient() {
```

Comment:
remove it in the iceberg_metadata_reader

Reply:
removed